x/delegation/keeper/un_delegation_state.go

```diff
@@ -150,6 +150,37 @@ func (k *Keeper) GetStakerUndelegationRecords(ctx sdk.Context, stakerID, assetID
 	return k.GetUndelegationRecords(ctx, recordKeys)
 }
 
+// IterateUndelegationsByStakerAndAsset iterate the undelegation records according to the stakerID and assetID.
+func (k *Keeper) IterateUndelegationsByStakerAndAsset(
+	ctx sdk.Context, stakerID, assetID string, isUpdate bool,
+	opFunc func(undelegation *types.UndelegationRecord) (bool, error),
+) error {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixStakerUndelegationInfo)
+	iterator := sdk.KVStorePrefixIterator(store, []byte(strings.Join([]string{stakerID, assetID}, "/")))
```

Comment:
use `delegationtype.GetDelegationStateIteratorPrefix(stakerID, assetID)` ?

Reply:
Right, I will rename the function `GetDelegationStateIteratorPrefix` and use it here.